targets/pdb/include/PinNames.h

```diff
@@ -148,12 +148,12 @@ typedef enum {
 #ifdef MBED_CONF_TARGET_STDIO_UART_TX
   STDIO_UART_TX = MBED_CONF_TARGET_STDIO_UART_TX,
 #else
-  STDIO_UART_TX = PB_3,  // JTDO
+  STDIO_UART_TX = (int)0xFFFFFFFF,
 #endif
 #ifdef MBED_CONF_TARGET_STDIO_UART_RX
   STDIO_UART_RX = MBED_CONF_TARGET_STDIO_UART_RX,
 #else
-  STDIO_UART_RX = PA_15,  // JTDI
+  STDIO_UART_RX = (int)0xFFFFFFFF,
```

Comment:
what is this saying? maybe NC is more appropriate? so on the PDB, we have the SWD interface only right? we no longer have that debug header?

Reply:
Ya it only has the SWD header